showkase/src/main/java/com/airbnb/android/showkase/ui/ShowkaseGroupsScreen.kt

```diff
@@ -0,0 +1,114 @@
+package com.airbnb.android.showkase.ui
+
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.navigation.NavHostController
+import com.airbnb.android.showkase.models.ShowkaseBrowserColor
+import com.airbnb.android.showkase.models.ShowkaseBrowserComponent
+import com.airbnb.android.showkase.models.ShowkaseBrowserScreenMetadata
+import com.airbnb.android.showkase.models.ShowkaseBrowserTypography
+import com.airbnb.android.showkase.models.ShowkaseCurrentScreen
+import com.airbnb.android.showkase.models.update
+import java.util.Locale
+
+@Composable
+internal fun ShowkaseGroupsScreen(
+    groupedTypographyMap: Map<String, List<*>>,
+    showkaseBrowserScreenMetadata: MutableState<ShowkaseBrowserScreenMetadata>,
+    navController: NavHostController,
+    onClick: () -> Unit
+) {
+    val filteredMap = getFilteredSearchList(
+        groupedTypographyMap.toSortedMap(),
+        showkaseBrowserScreenMetadata
+    )
+
+    val string = filteredMap.map { it.key }.joinToString()
```

Comment:
where is string used? a better name would be easier to follow

Reply:
not used anywhere..